region/maridia/outer/Mama Turtle Room.json

```diff
@@ -664,6 +664,24 @@
         "The easiest method is to jump over her by jumping on her babies, or to Morph under her."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "SpaceJump Escape with Walljump",
+      "requires": [
+        "canSuitlessMaridia",
+        "SpaceJump",
+        "canPreciseWalljump",
```

Comment:
```suggestion
        "canWalljump",
        "canCarefulJump",
```

Reply:
Soo... either of these formulations puts the strat in medium, which is what I think we want?

IDK, I formulated it the way I did because making sure you spin jump at the top is free, you can spin jump towards the wall.  But if you miss the walljump (which I have) Mama Turtle whacks you.

Reply:
My thought is that you have multiple chances to walljump before you reach the water.
I can see the tech needed here being focused on the walljump, so indicating it as a harder than base form walljump would be appropriate.  canCarefulJump isn't that related to what is happening in the strat but is used for general movement checks.
So you can leave it as it is.